packages/api/src/command/medical/document/document-download.ts

```diff
@@ -120,15 +122,22 @@ const doesObjExist = async ({
   }
 };
 
-export const getSignedURL = async ({ fileName }: { fileName: string }): Promise<string> => {
+export const getSignedURL = async ({
+  fileName,
+  bucketName,
+}: {
+  fileName: string;
+  bucketName?: string;
+}): Promise<string> => {
   const urlExpirationSeconds = URL_EXPIRATION_TIME.asSeconds();
 
   const url = s3client.getSignedUrl("getObject", {
     // TODO 760 Fix this
-    Bucket: Config.isSandbox()
-      ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        Config.getSandboxSeedBucketName()!
-      : Config.getMedicalDocumentsBucketName(),
+    Bucket:
+      bucketName ?? Config.isSandbox()
+        ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          Config.getSandboxSeedBucketName()!
+        : Config.getMedicalDocumentsBucketName(),
```

Comment:
not following the logic...

if `bucketName` is specified on sandbox, then we'll always use `Config.getSandboxSeedBucketName()!` ?

but if it's not on Sandbox, then we'll use `Config.getSandboxSeedBucketName()!` anyways?

Reply:
Fixed